allennlp/data/tokenizers/word_filter.py

```diff
@@ -33,37 +33,90 @@ def filter_words(self, words: List[Token]) -> List[Token]:
         return words
 
 
+@WordFilter.register('regex')
+class RegexFilter(WordFilter):
+    """
+    A ``RegexFilter`` removes words according to supplied regex patterns.
+
+    Parameters
+    ----------
+    patterns : ``List[str]``
+        List of regex patterns to filter by.
```

Comment:
I still feel like I have to think too much to figure out if these regexes will keep words or remove words.  Can you just say directly which it is?  Something like "Words matching these regex patterns will be removed as stopwords."  Also, you don't need the blank line at the end of the docstring.